### PR TITLE
OCPVE-603: Add lvms tag to volume groups

### PR DESCRIPTION
**Pull request by suleymanakbas91**
This PR adds an `lvms` tag to the volume groups used by LVMS. Also, this implements a temporary logic to add the tag to the existing volume groups, which is useful especially for upgrades.

### CODE REVIEW

File: cmd/vgmanager/vgmanager.go [hunk 1]
```diff
@@ -132,3 +151,52 @@ func run(_ *cobra.Command, _ []string, opts *Options) error {
 
 	return nil
 }
+
+// addTagToVGs adds a lvms tag to the existing volume groups. This is a temporary logic that should be removed in v4.16.
+func addTagToVGs(ctx context.Context, c client.Client, lvm *lvm.HostLVM, nodeName string, namespace string, logger logr.Logger) error {
```
Comment (jakobmoellerdev) [inline cmd/vgmanager/vgmanager.go:156]: if this fails, lvms will crash. is this desired behavior or should we simply continue?

Comment (jakobmoellerdev) [inline cmd/vgmanager/vgmanager.go:156]: test missing for this function

Comment (jakobmoellerdev) [inline cmd/vgmanager/vgmanager.go:156]:
```suggestion
func addTagToVGs(ctx context.Context, c client.Client, lvm lvm.LVM, nodeName string, namespace string) error {
   logger := log.FromContext(ctx)
```

**Review by jakobmoellerdev — COMMENTED** [review]
just some minor questions and missing test, other than that lgtm :)

File: cmd/vgmanager/vgmanager.go (hunk 1 above)
Comment (suleymanakbas91) [reply cmd/vgmanager/vgmanager.go:156]: Good point. We should continue in my opinion.

Comment (suleymanakbas91) [reply cmd/vgmanager/vgmanager.go:156]: I was thinking of covering this within QE testing, as this is temporary logic. Do you think we should add an e2e test for this?

Comment (suleymanakbas91) [reply cmd/vgmanager/vgmanager.go]: I've at least added a unit test.

**Review by jakobmoellerdev — APPROVED** [review]
/lgtm

**Pull request merged**